data/json/items/gun/bio.json

```
@@ -17,10 +17,10 @@
     "ranged_damage": { "damage_type": "bullet", "amount": 2 },
     "dispersion": 500,
     "durability": 6,
-    "clip_size": 4,
-    "modes": [ [ "DEFAULT", "single shot", 1 ], [ "burst", "double", 2 ] ],
-    "reload": 200,
-    "flags": [ "NEVER_JAMS", "NO_DROP", "RELOAD_EJECT", "NO_UNWIELD", "TRADER_AVOID", "RELOAD_ONE" ]
+    "clip_size": 8,
+    "modes": [ [ "DEFAULT", "single shot", 1 ], [ "burst", "double", 2 ], [ "AUTO", "auto", 4 ] ],
+    "reload": 1600,
+    "flags": [ "NEVER_JAMS", "NO_DROP", "RELOAD_EJECT", "NO_UNWIELD", "TRADER_AVOID" ]
```

```suggestion
    "clip_size": 8,
    "modes": [ [ "DEFAULT", "single shot", 1 ], [ "burst", "double", 2 ], [ "AUTO", "auto", 4 ] ],
    "reload": 200,
    "flags": [ "NEVER_JAMS", "NO_DROP", "RELOAD_EJECT", "NO_UNWIELD", "TRADER_AVOID", "RELOAD_ONE" ]
```
Worth noting that reload mechanics in BN are potat and this is going to get applied per round loaded. See for example why magic ranged weapons in Arcana have very low reload rates, because that adds up when reloading all of its ammo at once: https://github.com/chaosvolt/cdda-arcana-mod/blob/master/Arcana_BN/items/ranged.json#L197

Even if it did work the way you assumed it did, it'd be a lil odd to suffer the full reload duration if you only had a single shell to shove into it.